roadmap/implementers-guide/src/node/approval/approval-voting.md

```diff
@@ -2,6 +2,14 @@
 
 Reading the [section on the approval protocol](../../protocol-approval.md) will likely be necessary to understand the aims of this subsystem.
 
+Approval votes are split into two parts: Assignments and Approvals. Validators first broadcast their assignment to indicate intent to check a candidate. Upon successfully checking, they broadcast an approval vote. If a validator doesn't broadcast their approval vote shortly after issuing an assignment, this is an indication that they are being prevented from recovering or validating the block data and that more validators should self-select to check the candidate. This is known as a "no-show".
+
+The core of this subsystem is a Tick-based timer loop, where Ticks are 500ms. We also reason about time in terms of DelayTranches, which measure the number of ticks elapsed since a block was produced. We track metadata for all un-finalized but included candidates. We compute our local assignments to check each candidate, as well as which DelayTranche those assignments may be minimally triggered at. As the same candidate may appear in more than one block, we must produce our potential assignments for each (Block, Candidate) pair. The timing loop is based on waiting for assignments to become no-shows or waiting to broadcast and begin our own assignment to check.
```

Comment:
We selected this 500ms because it evenly divides the block time while being a generous margin over expected network round round trip times.  We do not need to record this reasoning here thought I guess.

Reply:
It doesn't evenly divide the block time for all possible chains. Our chain uses 6000ms which is fine.

Reply:
yes of course.  We maybe do not care so much about mapping them back to the relay chain clock anyways now that I think about it.